auth0/resource_auth0_user_test.go

```diff
@@ -173,63 +174,46 @@ func TestAccUser(t *testing.T) {
+					resource.TestCheckResourceAttr("auth0_user.user", "username", strings.ToLower(t.Name())),
 					resource.TestCheckResourceAttr("auth0_user.user", "user_id", fmt.Sprintf("auth0|%s", strings.ToLower(t.Name()))),
 					resource.TestCheckResourceAttr("auth0_user.user", "email", fmt.Sprintf("%s@acceptance.test.com", strings.ToLower(t.Name()))),
 					resource.TestCheckResourceAttr("auth0_user.user", "name", "Firstname Lastname"),
-					resource.TestCheckResourceAttr("auth0_user.user", "family_name", "Lastname"),
 					resource.TestCheckResourceAttr("auth0_user.user", "given_name", "Firstname"),
+					resource.TestCheckResourceAttr("auth0_user.user", "family_name", "Lastname"),
 					resource.TestCheckResourceAttr("auth0_user.user", "nickname", strings.ToLower(t.Name())),
-					resource.TestCheckResourceAttr("auth0_user.user", "connection_name", "Username-Password-Authentication"),
-					resource.TestCheckResourceAttr("auth0_user.user", "roles.#", "0"),
 					resource.TestCheckResourceAttr("auth0_user.user", "picture", "https://www.example.com/picture.jpg"),
+					resource.TestCheckResourceAttr("auth0_user.user", "user_metadata", `{"baz":"qux","foo":"bar"}`),
+					resource.TestCheckResourceAttr("auth0_user.user", "app_metadata", `{"baz":"qux","foo":"bar"}`),
+					resource.TestCheckResourceAttr("auth0_user.user", "roles.#", "0"),
 				),
 			},
 			{
-				Config: template.ParseTestName(testAccUserAddRole, strings.ToLower(t.Name())),
+				Config: template.ParseTestName(testAccUserUpdate, strings.ToLower(t.Name())),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr("auth0_user.user", "roles.#", "2"),
 					resource.TestCheckResourceAttr("auth0_role.owner", "name", "owner"),
 					resource.TestCheckResourceAttr("auth0_role.admin", "name", "admin"),
+					resource.TestCheckResourceAttr("auth0_user.user", "user_metadata", `{"foo":"bars"}`),
+					resource.TestCheckResourceAttr("auth0_user.user", "app_metadata", `{"foo":"bars"}`),
 				),
 			},
 			{
-				Config: template.ParseTestName(testAccUserRemoveRole, strings.ToLower(t.Name())),
+				Config: template.ParseTestName(testAccUserUpdateAgain, strings.ToLower(t.Name())),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr("auth0_user.user", "roles.#", "1"),
+					resource.TestCheckResourceAttr("auth0_user.user", "user_metadata", `{"foo":"barss","foo2":"bar2"}`),
+					resource.TestCheckResourceAttr("auth0_user.user", "app_metadata", `{"foo":"barss","foo2":"bar2"}`),
 				),
 			},
-		},
-	})
-}
-
-const testAccUserCanSerializeEmptyMetadataFields = `
-resource auth0_user auth0_user_issue_218 {
-  connection_name = "Username-Password-Authentication"
-  user_id = "id_{{.testName}}"
-  username = "user_{{.testName}}"
-  email = "issue.218.{{.testName}}@acceptance.test.com"
-  email_verified = true
-  password = "MyPass123$"
-}
-`
-
-func TestAccUserCanSerializeEmptyMetadataFields(t *testing.T) {
```

Comment:
We're removing this as we added it as a last test case for the TestAccUser scenario.